src/core/etl/src/Flow/ETL/DSL/To.php

```diff
@@ -18,14 +18,6 @@
  */
 class To
 {
-    /**
-     * @param int<1, max> $bufferSize
-     */
-    final public static function buffer(Loader $overflowLoader, int $bufferSize) : Loader
```

Comment:
This most likely also requires note in upgrade file.

Reply:
it's already there

Reply:
It says about `BufferLoader`, but not about the `To::buffer()` method, which is related but still different place.

Reply:
it's the same thing, To::buffer is only a DSL function to simplify initialization of BufferLoader but feel free to add it into the upgrade notes